pages/overview/contributing.md

```diff
@@ -119,6 +119,33 @@ pull request on the github.
 git push my_changes origin
 ~~~
 
+## Naming conventions in the documentation
```

Comment:
Maybe add some clarification for devices used.
We have:
- cache
- cache device
- exported object = cache volume (= *colloquially* core...)
- core device = backend device = core? 😮

Should core (by itself) exist in documentation?